client/src/amplify-ui/code-block-switcher/code-block.spec.ts

```diff
@@ -0,0 +1,25 @@
+import {AmplifyCodeBlockSwitcher} from "./code-block-switcher";
```

Comment:
Code block and code block switch tests should be different

Reply:
Good catch